### PR TITLE
CommunityToolkit docs for EventStore integration

### PR DESCRIPTION
**Pull request by fredimachado**
## Summary

CommunityToolkit docs for EventStore integration

Integration PR: https://github.com/CommunityToolkit/Aspire/pull/277

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/community-toolkit/hosting-eventstore.md](https://github.com/dotnet/docs-aspire/blob/f5884dee78ec2b59e1ffd4ac99533f26abbc5a74/docs/community-toolkit/hosting-eventstore.md) | [.NET Aspire Community Toolkit EventStore integration](https://review.learn.microsoft.com/en-us/dotnet/aspire/community-toolkit/hosting-eventstore?branch=pr-en-us-2143) |
| [docs/toc.yml](https://github.com/dotnet/docs-aspire/blob/f5884dee78ec2b59e1ffd4ac99533f26abbc5a74/docs/toc.yml) | [docs/toc](https://review.learn.microsoft.com/en-us/dotnet/aspire/toc?branch=pr-en-us-2143) |

### CODE REVIEW

File: docs/community-toolkit/hosting-eventstore.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+### Add EventStore resource with data volume
+
+To add a data volume to the EventStore resource, call the `Aspire.Hosting.EventStoreBuilderExtensions.WithDataVolume` method on the EventStore resource:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var eventstore = builder.AddEventStore("eventstore")
+                         .WithDataVolume();
```
Comment (IEvangelist) [inline docs/community-toolkit/hosting-eventstore.md:63]:
```suggestion
var eventstore = builder.AddEventStore("eventstore")
                        .WithDataVolume();
```

File: docs/community-toolkit/hosting-eventstore.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+### Add EventStore resource with log volume
+
+To add a log volume to the EventStore resource, call the `Aspire.Hosting.EventStoreBuilderExtensions.WithLogVolume` method on the EventStore resource:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var eventstore = builder.AddEventStore("eventstore")
+                         .WithLogVolume();
```
Comment (IEvangelist) [inline docs/community-toolkit/hosting-eventstore.md:102]:
```suggestion
var eventstore = builder.AddEventStore("eventstore")
                        .WithLogVolume();
```

**Comment by IEvangelist** [comment]
We also need this to be added to the toc.yml.

**Comment by fredimachado** [comment]
> We also need this to be added to the toc.yml.

Thanks @IEvangelist , I just pushed the suggested changes.

**Pull request merged**